fastlane_core/lib/fastlane_core/ui/errors/fastlane_not_counted_error.rb

```diff
@@ -0,0 +1,11 @@
+module FastlaneCore
+  class Interface
+    class FastlaneNotCountedFailure < FastlaneException; end
```

Comment:
Maybe we can come up with a better name? Maybe something like `FastlaneCommonException` or similar?